catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -283,10 +280,9 @@ public void setRemoteName(String remoteName) {
         this.remoteName = remoteName;
     }
 
-    private String getRegistryId(Metacard mcard) {
-        Attribute attr = mcard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID);
-        if (attr != null && attr.getValue() instanceof String) {
-            return (String) attr.getValue();
+    private String getRegistryId(Metacard metacard) {
```

Comment:
I felt like there was a spot or two where it made sense still, but I will check on it.